airflow/www_rbac/blueprints.py

```diff
@@ -17,19 +17,11 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-from flask import Markup, Blueprint, redirect
-import markdown
+from flask import Blueprint, redirect
 
 routes = Blueprint('routes', __name__)
 
 
 @routes.route('/')
 def index():
     return redirect('/home')
-
-
-@routes.route('/health')
-def health():
```

Comment:
I wonder why we modify the health function in the blueprint.py in the old UI while delete this function and move it to views.py?

Either way is fine, just want to understand why we do the same(both delete and move to views.py) for old/new UI?